### ISSUE TITLE
Add spell correction

### ISSUE THREAD

**Issue by kseo** [purescript/pursuit#239]
It would be nice if Pursuit could automatically suggest alternative search keywords when the search failed due to typos.

Inputting `mayba` shows

```
Did you mean `maybe`?
```

instead of

```
No results for "mayba", sorry.
```

Suggesting the symbols with 1 or 2 Levenshtein distance will be a good starting point.

I will submit a PR if this proposal is accepted.

**Comment by hdgarrood** [comment]
I agree, this would be lovely, but I think I'd rather just have them appear in the original search results (with a lower ranking, of course).

**Comment by kseo** [comment]
@hdgarrood That's also nice! I will start working on this.
